src/tutorial/tutorial.test.tsx

```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
+import '@testing-library/jest-dom';
+import { showModal } from "@openmrs/esm-framework";
+import Tutorial from "./tutorial";
+
+jest.mock('@openmrs/esm-framework', () => ({
+	showModal: jest.fn(),
+}))
```

Comment:
```suggestion
const mockShowModal = jest.mocked('showModal');
```

And then replace `showModal` in assertions with `mockShowModal`. See https://o3-docs.openmrs.org/docs/frontend-modules/unit-and-integration-testing#you-should-not-need-to-use-partial-mocks for more context about why you shouldn't need to use partial mocks.